tfhe/src/high_level_api/prelude.rs

```diff
@@ -3,6 +3,7 @@
 //!
 //! It is meant to be glob imported:
 //! ```
+//! #[allow(unused_imports)]
```

Comment:
can you add a `# ` (with the sapce) in front ? IIRC it's a way to hide the warnings in docstrings

https://blog.guillaume-gomez.fr/articles/2020-03-12+Guide+on+how+to+write+documentation+for+a+Rust+crate#Hiding-lines